### PR TITLE
Add link from /all to /more

### PR DESCRIPTION
**Pull request by rraue**
Fixes https://github.com/mozilla/bedrock/issues/9610

### CODE REVIEW

File: bedrock/firefox/templates/firefox/all-unified.html [hunk 1]
```diff
@@ -90,8 +90,10 @@ <h1 class="c-intro-heading">{{ ftl('firefox-all-choose-which-firefox') }}</h1>
         </div>
         <div class="c-support-links">
           <ul>
+            <li><a href="{{ url('firefox.more') }}">{{ ftl('learn-more-about-firefox') }}</a>
             <li><a href="{{ url('privacy.notices.firefox') }}">{{ ftl('firefox-all-firefox-privacy-notice') }}</a></li>
             <li><a href="https://support.mozilla.org/products/?utm_source=mozilla.org&utm_medium=referral&utm_campaign=need-help-link">{{ ftl('firefox-all-need-help') }}</a></li>
+            </li>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/all-unified.html:96]: The closing `<li>` here should be on line 93 above.

File: l10n/en/firefox/all.ftl [hunk 2]
```diff
@@ -8,6 +8,7 @@ firefox-all-check-the-system-requirements = Check the system requirements
 firefox-all-release-notes = Release notes
 firefox-all-source-code = Source code
 firefox-all-need-help = Need help?
+learn-more-about-firefox = Learn more about Firefox
```
Comment (alexgibson) [inline l10n/en/firefox/all.ftl:11]: String Id should start with `firefox-all-`, and Firefox should use a brand term
```suggestion
firefox-all-learn-more-about-firefox = Learn more about { -brand-name-firefox }
```

**Review by alexgibson — CHANGES_REQUESTED** [review]

File: bedrock/firefox/templates/firefox/all-unified.html [hunk 3]
```diff
@@ -90,8 +90,10 @@ <h1 class="c-intro-heading">{{ ftl('firefox-all-choose-which-firefox') }}</h1>
         </div>
         <div class="c-support-links">
           <ul>
+            <li><a href="{{ url('firefox.more') }}">{{ ftl('learn-more-about-firefox') }}</a>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/all-unified.html:93]:
```suggestion
            <li><a href="{{ url('firefox.more') }}">{{ ftl('firefox-all-learn-more-about-firefox') }}</a>
```

**Comment by rraue** [comment]
Wont fix

**Pull request closed without merging**